src/packages/text_search.gleam

```diff
@@ -46,45 +48,113 @@ pub fn update(
   insert(index, name, description)
 }
 
+pub type LookupOutcome {
+  Packages(packages: List(String))
+  DidYouMean(suggestion: String)
```

Comment:
Make this a second function please, rather than always forcing this second scan in the event of no results.

Reply:
I think the comment is pointing to the wrong section of code: it's pointing to a type definition so I don't see what should be turned into a function 😅